app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -64,10 +63,18 @@ class AdministratorControlsViewModel @Inject constructor(
   private fun processAdministratorControlsList(
     deviceSettings: DeviceSettings
   ): List<AdministratorControlsItemViewModel> {
+//    uncomment following code for general options and add corresponding test cases
+//    val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
+//      AdministratorControlsGeneralViewModel()
+//    )
+//    itemViewModelList.add(
+//      AdministratorControlsProfileViewModel(
+//        routeToProfileListListener,
+//        loadProfileListListener
+//      )
+//    )
```

Comment:
@BenHenning Is this fine to comment this code to not show `Edit Accounts` option in `Administrator Controls` or we should just remove the code completely?

Reply:
I think we can remove this code completely instead of commenting it.

Reply:
@BenHenning

Reply:
The preferred way to do this is actually to use a platform parameter & gate it. We actually want this functionality, we just aren't ready to enable it yet.

You can create a new parameter by referencing https://github.com/oppia/oppia-android/blob/develop/domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterModule.kt#L41 (see how EnableLanguageUi is used since it's similar in that it disables some UI functionality behind a flag). @JishnuGoyal you should create a new flag that does the same thing here (disables the edit accounts option for admins).